lib/commands/start.js

```diff
@@ -62,22 +62,27 @@ class StartCommand extends Command {
                     if (!argv.quiet) {
                         const isInstall = process.argv[2] === 'install';
                         let adminUrl = instance.config.get('admin.url') || instance.config.get('url');
-
                         // Strip the trailing slash and add the admin path
                         adminUrl = `${adminUrl.replace(/\/$/,'')}/ghost/`;
 
-                        this.ui.log(`You can access your publication at ${chalk.cyan(instance.config.get('url'))}`, 'white');
+                        // Show a warning about direct mail - but in grey, it's not the most important message here
+                        if (isInstall && instance.config.get('mail.transport') === 'Direct') {
+                            this.ui.log('\nGhost uses direct mail by default. To set up an alternative email method read our docs at https://ghost.org/mail', 'gray');
+                        }
+
+                        if (isInstall && instance.config.get('process') !== 'local' && !urlUtils.isCustomDomain(instance.config.get('url'))) {
+                            // Special message for when a production install is completed with an ip or localhost instead of a domain
+                            this.ui.log('\n------------------------------------------------------------------------------', 'white');
+                            this.ui.log('To add a custom domain and secure your site, run', 'ghost setup ssl --url example.com', 'white', 'cmd');
```

Comment:
Hmmm @acburdine I just tested, and this command doesn't work

`ghost setup ssl --url https://myblogurl.com`

Results in

> SSL certs cannot be generated for IP addresses, skipping
ℹ Setting up SSL [skipped]

Tried with and without the equals sign.

Reply:
To get it to work I had to do:

`ghost config url https://myblogurl.com`

`ghost setup nginx ssl`

ghost setup ssl on its own said:

> Nginx config file does not exist, skipping SSL setup
ℹ Setting up SSL [skipped]

Reply:
right...so when you run `ghost setup` with individual steps it bypasses the "configuration" part of things, and just runs the individual steps.

Reply:
should be able to fix that with the setup refactor I think